docs/release-notes/8.5.asciidoc

```diff
@@ -16,6 +23,13 @@
 [[release-notes-8.5.2]]
 === 8.5.2
 
+[discrete]
+[[known-issue-8.5.2]]
+==== Known issues
+* The rule details page and **Edit rule settings** page load indefinitely if you edit a rule that has the `saved_id` property configured. All rule types, except for the custom query rule, are affected.
```

Comment:
nit: if we highlight **Edit rule settings*, should we do the same for "rule details"?

Reply:
Good question! So when we refer to these pages in the docs, we don't use the same text styling (bolding) because we're referring to them in different ways. 

When you edit a rule, the page that you encounter is literally named "Edit rule settings". Because the page has a very specific name, and the name represents a component that users interact with, we bold the page name in the docs. 

When you view a rule's details page, the page title doesn't literally reflect the "rule details page" component (i.e., it's not named "Rule details"). Instead, it matches the rule's name -- which is probably something that's set in the rule details page code. Since the rule details page title is dynamic, we reference the page/component more generally in the docs and don't bold the text. 

Hopefully that helps answer your question, but I'm happy to elaborate more on our styling guidelines if you have more :)

Reply:
Great attention to details, regarding page names. Thanks for shedding a light on this.